api/src/routes/guilds/#guild_id/vanity-url.ts

```diff
@@ -9,11 +9,19 @@ const InviteRegex = /\W/g;
 
 router.get("/", route({ permission: "MANAGE_GUILD" }), async (req: Request, res: Response) => {
 	const { guild_id } = req.params;
+	const guild = await Guild.findOneOrFail({ id: guild_id });
+
+	if (!guild.features.includes("ALIASABLE_NAMES")) {
```

Comment:
aliasable names need not be feature gated

Reply:
What should it be? I understood that you wanted a GuildFeatures flag